whetstone/runtime/src/main/java/com/freeletics/mad/whetstone/NavEntryComponent.kt

```diff
@@ -43,7 +43,7 @@ import kotlin.reflect.KClass
  * It's recommended to put this annotation on the method that provides the matching [NavEntryId]
```

Comment:
This comment isn't relevant anymore

Reply:
[4d5631f](https://github.com/freeletics/mad/pull/71/commits/4d5631fae0f23bb55ebe10e6f01674084e88f84e)